apps/hash-graph/bin/cli/src/subcommand/snapshot.rs

```diff
@@ -58,54 +60,25 @@ pub async fn snapshot(args: SnapshotArgs) -> Result<(), GraphError> {
 
     match args.command {
         SnapshotCommand::Dump(_) => {
-            let data_types = Read::<test_graph::OntologyTypeRecord<DataType>>::read(
-                &store,
-                &Filter::All(vec![]),
-                None,
-            )
-            .await
-            .change_context(GraphError)?;
-
-            let property_types = Read::<test_graph::OntologyTypeRecord<PropertyType>>::read(
-                &store,
-                &Filter::All(vec![]),
-                None,
+            serde_json::to_writer(
+                &mut std::io::stdout(),
+                &store.read_test_graph().await.change_context(GraphError)?,
             )
-            .await
+            .into_report()
             .change_context(GraphError)?;
-
-            let entity_types = Read::<test_graph::OntologyTypeRecord<EntityType>>::read(
-                &store,
-                &Filter::All(vec![]),
-                None,
-            )
-            .await
-            .change_context(GraphError)?;
-
-            let entities = Read::<Entity>::read(&store, &Filter::All(vec![]), None)
-                .await
-                .change_context(GraphError)?
-                .into_iter()
-                .map(test_graph::EntityRecord::from)
-                .collect();
-
-            let test_graph = test_graph::TestData {
-                block_protocol_module_versions: BlockProtocolModuleVersions {
-                    graph: semver::Version::new(0, 3, 0),
-                },
-                data_types,
-                property_types,
-                entity_types,
-                entities,
-                custom: test_graph::CustomGlobalMetadata::default(),
-            };
-
-            serde_json::to_writer(&mut std::io::stdout(), &test_graph)
+            writeln!(std::io::stdout())
                 .into_report()
                 .change_context(GraphError)?;
-            writeln!(std::io::stdout())
+        }
+        SnapshotCommand::Restore(_) => {
+            let test_data: test_graph::TestData = serde_json::from_reader(std::io::stdin())
```

Comment:
I think the `test_` prefix for the export is a little strange. Would it make more sense to just call it `snapshot`?

Reply:
Good point, I renamed it in [Rename variable to `snapshot`](https://github.com/hashintel/hash/pull/2327/commits/50eda2236b4634982c8636809d5df281c1238d22)

Reply:
I just saw in the other thread Alfie also pointed out the module structure for `test_graph`. I would agree that `snapshot` would make more sense. Don't mind it being a follow-up

Reply:
I like `snapshot` as module name as well. Will use that name in the follow-up then 🙂